waterloop-site/src/components/RecruitmentForm/Form.tsx

```diff
@@ -0,0 +1,200 @@
+    }
+    else if (event.currentTarget.name === "applicationTerm" && event.currentTarget.id === "3"){
+      this.setState({termCheckedArray: [false, false, false, true, false]});
+      this.setState({applicationTerm: this.props.termList[3]});
+    }
+    else if (event.currentTarget.name === "applicationTerm" && event.currentTarget.id === "4"){
+      this.setState({termCheckedArray: [false, false, false, false, true]});
+      this.setState({applicationTerm: this.props.termList[4]});
+    }
+    else if (event.currentTarget.name === "termType" && event.currentTarget.id === "a"){
+      this.setState({termTypeCheckedArray: [true, false, false]});
+      this.setState({termType: "School Term"});
+    }
+    else if (event.currentTarget.name === "termType" && event.currentTarget.id === "b"){
+      this.setState({termTypeCheckedArray: [false, true, false]});
+      this.setState({termType: "Work/Off Term in Waterloo"});
+    }
+    else if (event.currentTarget.name === "termType" && event.currentTarget.id === "c"){
+      this.setState({termTypeCheckedArray: [false, false, true]});
+      this.setState({termType: "Not in Waterloo"});
+    }
+    return;
+  }
+
+  handleTextAreaChange(event: React.FormEvent<HTMLTextAreaElement>){
+    if (event.currentTarget.name === "entry.288252419"){ // whyJoin
+            this.setState({whyJoin: event.currentTarget.value});
+    }
+    else if (event.currentTarget.name === "entry.1372043873"){
+            this.setState({technicalAns: event.currentTarget.value});
+    }
+    return;
+  }
+
+  render() {
+    return (
+      <div style = {{textAlign: 'center'}}>
+        <Block>
+          <InputForm action = "https://docs.google.com/forms/u/0/d/e/1FAIpQLSdmZSfD1Hs0D3MLBjAfdUkaCb3GJJxIvKUEVJCBf5hVxZTt4g/formResponse"
+            target="_self" method="POST">
+            <Header>Applying for {this.props.role.toUpperCase()}</Header>
+            <ShortInput style = {{display: 'none'}} value = {this.props.role} name = "entry.1805132656" placeholder="Position" readOnly/>
+
+            <ShortInput type="text" name = "entry.1105765972" placeholder="Full Name" onChange = {this.handleChange.bind(this)} required/>
+            <ShortInput type="text" name = "entry.204636100" placeholder="Email" onChange = {this.handleChange.bind(this)} required/>
+            <ShortInput type="text" name = "entry.1369449321" placeholder="Program" onChange = {this.handleChange.bind(this)} required/>
+            <ShortInput type="text" name = "entry.1488954434" placeholder="Your Current Term (ex. 2A)" onChange = {this.handleChange.bind(this)} required/>
+
+            <hr/>
+            <label style = {{display: 'block', margin: '1%'}}><b>Term you're applying for:</b></label>
+            <label style = {radioStyle}>
+              <input type = "radio" checked = {this.state.termCheckedArray[0]} id = "0" onChange = {this.handleChange.bind(this)} name = "applicationTerm" required/>
+            {this.props.termList[0]}</label>
```

Comment:
Common practice is to give the closing tag of the wrapping element it's own line, and then have the children all match in their indentation.
Also, in JSX, do not put spaces in-between a prop name, the equals sign, and the value
```
<label style = {radioStyle}>
  <input type="radio" checked={this.state.termCheckedArray[0]} id="0" onChange=this.handleChange.bind(this)} name="applicationTerm" required/>
  {this.props.termList[0]}
</label>
```